salt/utils/win_functions.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+'''
+Various functions to be used by windows during start up and to monkey patch
+missing functions in other modules
+'''
+from salt.exceptions import CommandExecutionError
+
+# Import Salt Libs
+import salt.utils
+
+# Import 3rd Party Libs
+try:
+    import ntsecuritycon
+    import psutil
+    import pywintypes
+    import win32api
+    import win32net
+    import win32security
+    HAS_WIN32 = True
+except ImportError:
+    HAS_WIN32 = False
+
+
+def __virtual__():
+    '''
+    Load only on Windows with necessary modules
+    '''
+    if not salt.utils.is_windows():
+        return False, 'This utility only works on Windows'
+    if not HAS_WIN32:
+        return False, 'This utility requires pywin32'
+
+    return 'win_runas'
```

Comment:
I am curious, is there a reason why this is 'win_runas' and not 'win_functions'?